subschedule.go

```diff
@@ -48,6 +48,7 @@ type SubscriptionScheduleInvoiceSettingsParams struct {
 type SubscriptionScheduleDefaultSettingsParams struct {
 	Params                `form:"*"`
 	ApplicationFeePercent *float64                                   `form:"application_fee_percent,high_precision"`
+	AutomaticTax          *SubscriptionAutomaticTaxParams            `form:"automatic_tax"`
```

Comment:
Technically this one should not "share" with the `Subscription` version and it should be its own class. But I know with go we've done a lot of mix and match to avoid duplicating all classes so I'm not blocking.